src/site/content/en/blog/share-like-a-native/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Share like a native app
+subhead: Web apps can use the same system provided share capabilities as native apps.
+authors:
+  - joemedley
+date: 2019-11-08
+hero: hero.jpg
+alt: A hand sharing a web.dev article.
+description: |
+  With the Web Share API, web apps are able to use the same system provided
+  share capabilities as native apps. The Web Share API makes it possible for web
+  apps to share links, text, and files to other apps installed on the device in
+  the same way as native apps.
+tags:
+  - post
+  - capabilities
+---
+
+With the Web Share API, web apps are able to use the same system provided share
+capabilities as native apps. The Web Share API makes it possible for web apps to
+share links, text, and files to other apps installed on the device in the same
+way as native apps.
+
+<figure class="w-figure w-figure--inline-right">
+  <img src="./wst-send.png" style="max-width: 25vw;" alt="System-level share target picker with an installed PWA as an option."/>
```

Comment:
Non-blocking, but might be good to highlight the PWA since there are so many apps in the screenshot.